### PR TITLE
Allowed WithBuildingExplosion to randomly select from multiple sequences

### PR DESCRIPTION
**Pull request by reaperrr**
And used that to implement TS building explosions matching the original.

### CODE REVIEW

File: OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs [hunk 1]
```diff
@@ -1679,6 +1679,13 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 							raceNode.Key = "ForceFaction";
 					}
 				}
+
```
Comment (Mailaender) [inline OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs]: > [SP2000] Invalid spacing at the end of the line.

**Comment by reaperrr** [comment]
Updated.

**Comment by Mailaender** [comment]
Nice polishing especially for the larger buildings. :+1:

**Comment by GraionDilach** [comment]
:+1:

**Comment by abcdefg30** [comment]
Looks also good to me. :+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/9d442967474fbe1c07617b9d67ac85f08992dddc...53cf9948e0338c33f2801d4c70bebb520e118e3d)

**Pull request merged**